lib/pacemaker/pcmk_injections.c

```diff
@@ -485,55 +494,55 @@ modify_configuration(pe_working_set_t * data_set, cib_t *cib, pcmk_injections_t
         CRM_ASSERT(rc == pcmk_ok);
     }
 
-    if (injections->watchdog) {
+    if (injections->watchdog != NULL) {
         rc = update_attr_delegate(cib, cib_sync_call | cib_scope_local,
-                             XML_CIB_TAG_CRMCONFIG, NULL, NULL, NULL, NULL,
-                             XML_ATTR_HAVE_WATCHDOG, injections->watchdog, FALSE, NULL, NULL);
-
+                                  XML_CIB_TAG_CRMCONFIG, NULL, NULL, NULL, NULL,
+                                  XML_ATTR_HAVE_WATCHDOG, injections->watchdog,
+                                  FALSE, NULL, NULL);
         CRM_ASSERT(rc == pcmk_ok);
     }
 
-    for (gIter = injections->node_up; gIter != NULL; gIter = gIter->next) {
-        char *node = (char *)gIter->data;
+    for (iter = injections->node_up; iter != NULL; iter = iter->next) {
+        char *node = (char *) iter->data;
 
         out->message(out, "inject-modify-node", "Online", node);
 
         cib_node = pcmk__inject_node_state_change(cib, node, true);
         CRM_ASSERT(cib_node != NULL);
 
         rc = cib->cmds->modify(cib, XML_CIB_TAG_STATUS, cib_node,
-                                      cib_sync_call | cib_scope_local);
+                               cib_sync_call | cib_scope_local);
```

Comment:
In #2598, you also got rid of the spaces around the pipe in expressions like this.

Reply:
Updated commit (and rebased on current master). We don't have a guideline for this, so it doesn't really matter, but I find that these expressions tend to get long and skipping the spaces makes it more likely to fit on a line.